src/RtspConnection.c

```diff
@@ -724,7 +724,7 @@ static bool parseUrlAddrFromRtspUrlString(const char* rtspUrlString, char* desti
 
 // SDP attributes are in the form:
 // a=x-nv-bwe.bwuSafeZoneLowLimit:70\r\n
-bool parseSdpAttributeToUInt(const char* payload, const char* name, unsigned int* val) {
+bool parseSdpAttributeToUInt(const char* payload, const char* name, uint32_t* val) {
```

Comment:
Is `unsigned int` 16 bits on 3DS or something?

Reply:
I'm debugging it but it seems that yes with only 3DS devkitpro installed. Don't know how it passed with WiiU, it may be this different...